app/components/ui/CalculatorInput.tsx

```diff
@@ -234,7 +333,12 @@ const CalculatorInput = () => {
       </div>
     );
   } else if (view === "dashboard") {
-    return <Dashboard data={data as Household} />;
+    return (
+      <Dashboard
+        processedData={data as Household}
+        inputData={formData as formType}
```

Comment:
See previous comment - I think the below should work.

```suggestion
        inputData={methods.getValues()}
```